libs/designsystem/src/lib/directives/modal-router-link/modal-router-link.directive.ts

```diff
@@ -9,10 +10,15 @@ export class ModalRouterLinkDirective {
   constructor(private modalNavigationService: ModalNavigationService) {}
 
   @Input('kirbyModalRouterLink') path: string | string[];
+  // tslint:disable-next-line:no-input-rename
+  @Input('kirbyModalQueryParams') queryParams: Params | null;
```

Comment:
Why not an optional input? 🤔
```suggestion
  @Input('kirbyModalQueryParams') queryParams?: Params;
```